### ISSUE TITLE
Tutorial when first using Me-app

### ISSUE THREAD

**Issue by jamalv** [teamforus/general#205]
Learn more about change requests here: https://bit.ly/39CWeEE

### Requested by:
@GerbenBosschieter 

### Change description
As a user i would like to understand how to use the me app after i first download it

**Comment by maxvisser** [comment]
You mean like a welcome screen with multiple images to explain the basic concept?

**Comment by jamalv** [comment]
Max, what do you think?

**Comment by jamalv** [comment]
Yea!

**Comment by Myshelova** [comment]
We can make onboarding screens and show them before registration flow

**Comment by jamalv** [comment]
I think we have two or three options here:

1. Show multiple screens about the app before the registration flow
Since someone has already looked at the marketing page in the app store I think they have already made the decision to use the app. Therefore I don't think this will be really necessary.

2. Show after signup / login
Here someone has signed up and might be curious about what to do next and what's possible with the app. I think it would be nice to make a good info page that is automatically opened the first time someone is logged in, and that they can also open the next time they log in.

3. Both before and after
We can also add a simple welcome page before the registration flow, with some basic important info, and then also add an info page within the app when you are logged in. 

We should think about what you would like to know as a user

**Comment by maxvisser** [comment]
I think number two is already implemented (on android atleast) as in we can just add the screens with the info; Low hanging fruit.

**Comment by GerbenBosschieter** [comment]
Just some onboarding screens for now would be nice.

What about adding more context about the usage of the app itself.
With a nice succes sentence after first login (congratz, welcome to me)
Added with some informatio as:
- Are you part of an organization? Use this app as your cashier, etc ect etc
- Have you received a voucher? Manage your vouchers with Me, etc etc

**Comment by jamalv** [comment]
Yea, maybe add a new welcome screen before signup, and adding a couple screens that are opened after the signup flow and that can be re-opened?

Like this: https://www.figma.com/file/ARUlmFPD1ZHOu5LZEgAY6P/%F0%9F%94%B5Me-Info-button?node-id=16%3A65

**Comment by jamalv** [comment]
@maxvisser I used the file you created for your proposal, I think you forgot to link it here?

**Comment by maxvisser** [comment]
This is the proposal for the info button

**Comment by maxvisser** [comment]
I see what you did now. Yeah thats also an idea.. Personally I don't like it but maybe others do.

**Comment by jamalv** [comment]
Hmm, yea, I think that people do understand this (they also did it during signupflow) so some information might be more useful. Also curious what @GerbenBosschieter thinks about both proposals.

(they are two pages in the linked figma file)

**Comment by maxvisser** [comment]
Don't get me wrong I like the tutorial but would do it just on signup or add it on the 'about Me' page in settings

**Comment by GerbenBosschieter** [comment]
I don't think that context about the QR-code self is needed as you proposed in Figma. @maxvisser . 

And I like something what you did jamal, some clean pages with general context. But I am not sure what the best moment is to show. Maybe not including these pages always in the flow.

Maybe the succes page after signup could have link like: Wanna know what you can do with Me? -> go to the 'tutorial' pages.

**Comment by jamalv** [comment]
Hahaha, "what you can do with me" 😂 

Yea, the way I see it the info pages open after the signup is complete. Then you have the option to read the pages, or to click them away (x at the top) if you then want to read them at a later time, you can click the little (i) icon on the scanner to open them again.

**Comment by GerbenBosschieter** [comment]
You can do a lot with me ;)

Is the QR scanner the right screen to add the info icon? Isn't it a bit weird. I think it's more logic to add it to the profile screen?

Just some minor things. I am okay with implementing it like you recommended.

**Comment by jknnr** [comment]
Closing this cause we have a new one that is more specific https://github.com/teamforus/me/issues/443

**Comment by jamalv** [comment]
Re-opening to have a discussion and get some feedback

**Comment by maxvisser** [comment]
added profile page and about me page to figma prototype; Would like to aswell update these. I also would say 'about me' would be nice to open the onboarding screens.

**Comment by maxvisser** [comment]
@jamalv whats the next step?

**Comment by maxvisser** [comment]
This is done on Me-android; we will show it during sprint demo. Release will be planned next week
